Source/WebKit/NetworkProcess/cocoa/NetworkDataTaskCocoa.mm

```diff
@@ -387,6 +387,15 @@ static float toNSURLSessionTaskPriority(WebCore::ResourceLoadPriority priority)
 
 void NetworkDataTaskCocoa::didCompleteWithError(const WebCore::ResourceError& error, const WebCore::NetworkLoadMetrics& networkLoadMetrics)
 {
+    if (networkLoadMetrics.responseBodyBytesReceived > totalBytesTransferredOverNetwork()) {
+        WTFEmitSignpost(m_task.get(), DataTask, "received 0 bytes");
```

Comment:
This doesn't seem to be the right string to associate with this signpost interval.

I think I'd leave out the `WTFEmitSignpost` call entirely here.

Reply:
But it may call `didReceiveData()` so treating it in part of completed sounds wrong.